features/bootstrap/ContactContext.php

```diff
@@ -49,22 +49,11 @@ public function iCompleteTheForm()
     public function iShouldSeeAThankYouMessage()
     {
         $this->assertSession()
-            ->elementContains('css', '.info-bar--success', 'Thanks Foo Bar, we have received your question.');
+            ->elementContains('css', '.info-bar--success', 'Thank you for your question. We will respond as soon as we can.');
```

Comment:
As requested per discussion. Change 'should' to 'will' as should makes it look like it could or could not work.